package.json

```diff
@@ -18,8 +18,10 @@
     "test:bun": "bun run deploy/bun.ts",
     "test:template": "npm run --tag test --template",
     "-------------------------------------------------": "",
-    "build": "rimraf lib && tsc && rollup -c",
+    "build": "rimraf lib && tsc && tsc -p ./tsconfig.cli.json && rollup -c",
```

Comment:
in CJS environment, generating declarations fails for the cleye library, so I split the build command